src/main/java/trello/entities/List.java

```diff
@@ -94,7 +94,7 @@ public void setCards(final ArrayList<Card> cards) {
      * @param listCards is the list of cards to compare with.
      * @return true if the ArrayList is equal to this list's cards.
      */
-    public boolean areListsEquals(final ArrayList<Card> listCards) {
+    public boolean areListsEquals(final java.util.List<Card> listCards) {
```

Comment:
Please go up this import to the section of import.

Reply:
This is not an import. It is like that so the IDE couldn't confuse between the java.util.List and the entity List that we are using in the context.